lcls-twincat-motion/Library/POUs/Motion/States/FB_PositionStateMove.TcPOU

```diff
@@ -91,17 +91,25 @@ VAR_OUTPUT
 END_VAR
 VAR
     fbMotionRequest: FB_MotionRequest;
+    rtExec: R_TRIG;
+    rtReset: R_TRIG;
+    bInnerExec: BOOL;
     bAllowMove: BOOL;
+    nLatchAllowErrorID: UDINT;
 END_VAR
 ]]></Declaration>
     <Implementation>
       <ST><![CDATA[// Veto the move for uninitialized and unsafe states
 bAllowMove := stPositionState.bMoveOk AND stPositionState.bValid AND stPositionState.bUpdated;
 
+rtExec(CLK:=bExecute);
+bInnerExec S= rtExec.Q AND bAllowMove;
+bInnerExec R= NOT bExecute;
```

Comment:
I'd naively have expected if there were a latched error we wouldn't execute `fbMotionRequest` at all. I'd imagine there are some details I'm missing there - mind giving a quick explanation?

Reply:
I think, despite your other comment, you're correct. It may be worth doing something additional here, making the inner function blocks also protect themselves for the sake of robustness.

Reply:
Handled in https://github.com/pcdshub/lcls-twincat-motion/pull/205/commits/cc683b2eff1adc02ef9082efaa88601d23c5522e